container/Dockerfile

```diff
@@ -0,0 +1,87 @@
+# Courtesy of Federico Ficarelli
+
+FROM ghcr.io/pulp-platform/snitch_cluster:latest@sha256:9978f9d0e3e6b7ad1b2c1f0eb628a113226e8443bc42ae8b4ddd3f34d9528e96 as builder
+
+RUN git clone --recursive https://github.com/kuleuven-micas/snitch_cluster.git /src \
+ # verilator model
+ && cd /src/target/snitch_cluster \
+ && make bin/snitch_cluster.vlt
```

Comment:
Thanks for reporting, I'll investigate

Reply:
The reason why this failed was because we updated the main branch of the repository, and the Dockerfile still linked to an old starter container, new versions are fixed now